examples/finetune/finetune.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+cd `dirname $0`
+cd ../..
+
+EXE="./finetune"
+
+# MODEL="c:/models/openllama-3b-v2-q8_0.gguf" # This is the model the readme uses.
+MODEL="c:/models/openllama-3b-v2.gguf" # An f16 model. Note in this case with "-g", you get an f32-format .BIN file that isn't yet supported if you use it with "main --lora" with GPU inferencing.
```

Comment:
Hardcoding an absolute Windows path inside of a _bash_ script (Unix-like) is an assumption that I think will not apply to many users. Better to use a relative path.

Same issue with --train-data below.

Reply:
I added variables for this. (This was an adventure, ending in filing a [w64devkit issue](https://github.com/skeeto/w64devkit/issues/90)).